src/QMCWaveFunctions/LCAO/RadialOrbitalSetBuilder.h

```diff
@@ -240,8 +240,9 @@ bool RadialOrbitalSetBuilder<COT>::addGridH5(hdf_archive& hin)
     ri = tt;
     hin.read(tt, "grid_rf");
     rf = tt;
-    hin.read(tt, "rmax_safe");
-    rmax_safe = tt;
+    // Ye to Anouar: do we need this input? It is missing in current h5
+    //hin.read(tt, "rmax_safe");
+    //rmax_safe = tt;
```

Comment:
No this should actually be moved outside. no grid should be hard coded in HDF5 as there is no way of setting it properly without hard coding it.

Reply:
I need to change it as TODO. This PR has not touched adding back the grid control in the h5 route.